tests/e2e/cli/detection/test_detection.py

```diff
@@ -222,7 +222,7 @@ def test_otx_deploy_openvino(self, template, tmp_dir_path):
     @pytest.mark.parametrize("template", templates, ids=templates_ids)
     def test_otx_eval_deployment(self, template, tmp_dir_path):
         tmp_dir_path = tmp_dir_path / "detection"
-        otx_eval_deployment_testing(template, tmp_dir_path, otx_dir, args, threshold=0.0)
+        otx_eval_deployment_testing(template, tmp_dir_path, otx_dir, args, threshold=0.05)
```

Comment:
Does deployment change openvino IR's performance? I think it should be same before and after deployment since it is same openvino.xml.

Reply:
Nice opinion, we could you 0.0 threshold for the deployment testing

Reply:
Done. https://github.com/openvinotoolkit/training_extensions/pull/1988/commits/838c5364f77ae4213df5a1aacfa119b0e79d179e